coeus-impl/src/main/java/org/kuali/kra/protocol/ProtocolDocumentBase.java

```diff
@@ -206,7 +205,16 @@ else if (isDisapproved(statusChangeEvent)) {
         }
     }
     
+    protected DocumentRouteHeaderValue getDocumentRouteHeaderValue() {
+        return KcServiceLocator.getService(RouteHeaderService.class).getRouteHeader(
+                this.getDocumentHeader().getWorkflowDocument().getDocumentId());
+    }
   
+    protected void setProtocolDocumentToApproveByDefault() {
+        DocumentRouteHeaderValue document = getDocumentRouteHeaderValue();
+        document.getDocumentType().getDefaultApprovePolicy().setPolicyValue(true);
+    }
```

Comment:
It is only for the merged Protocol document (amendment/renewal) where we need to blanket approve the new protocol document.